apps/flipper/pubspec.yaml

```
@@ -2,10 +2,10 @@ name: flipper_rw
 description: "A new Flutter project."
 publish_to: "none" # Remove this line if you wish to publish to pub.dev
 
-version: 1.170.4252223233643+1744398697
-
+version: 1.170.4252223233815+1745395006
 
 
+# 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)